packages/react/src/card/Card.tsx

```diff
@@ -52,4 +85,85 @@ const Card = ({
   );
 };
 
-export { Card, type CardProps };
+type RACLinkProps = Pick<LinkProps, 'href' | 'routerOptions' | 'children'>;
+
+type CardLinkWrapperProps = {
+  [Key in keyof RACLinkProps]: LinkProps[Key];
+} & {
+  // Override children type of LinkProps as it also allows a callback which is not allowed in HTMLProps
+  children: React.ReactNode;
+};
+
+type CardLinkProps = {
+  className?: string;
+} & (RACLinkProps | CardLinkWrapperProps);
+
+const cardLinkVariants = cva({
+  base: [
+    'cursor-pointer',
+    // **** Clickarea ****
+    'after:absolute',
+    'after:inset-[calc(theme(borderWidth.DEFAULT)*-1)]',
+    'after:rounded-[calc(theme(borderRadius.2xl)-theme(borderWidth.DEFAULT))]',
+  ],
+  variants: {
+    withHref: {
+      true: [
+        // **** Focus ****
+        'focus-visible:outline-none',
+        'focus-visible:after:outline-focus',
+        'focus-visible:after:outline-offset-2',
+        // **** Hover ****
+        // Links are underlined by default, and the underline is removed on hover.
+        // So we make sure that also happens when the user hovers the clickable area.
+        'hover:no-underline',
+      ],
+      false: [
+        // **** Focus ****
+        '[&_a:focus-visible]:outline-none',
+        '[&:has(a:focus-visible)]:after:outline-focus',
+        '[&:has(a:focus-visible)]:after:outline-offset-2',
+        // **** Hover ****
+        // Links are underlined by default, and the underline is removed on hover.
+        // So we make sure that also happens when the user hovers the card.
+        // The group-hover ensures that the hover effect also applies when this component is used as a wrapper around a link.
+        '[&_a]:group-hover/card:no-underline',
+      ],
+    },
+  },
+});
+
+/**
+ * A component that creates a clickable area on a card.
+ * It can be used either as a wrapper around a link or as a standalone link.
+ */
+const CardLink = ({
```

Comment:
Burde vi kalt denne `CardClickArea` istedenfor? 🤔 Eller skulle man hatt to komponenter en `CardLink` der `href` er påkrevd og en `CardClickArea` der `children` er påkrevd?